machine.go

```diff
@@ -363,10 +363,19 @@ func (m *Machine) writerKernelModules(w *writerhelper.WriterHelper) error {
 		"modules.builtin.bin",
 		"modules.devname"}
 
+	usrpath := "/lib/modules"
+	if mergedUsrSystem() {
+		usrpath = "/usr/lib/modules"
+	}
+	data, err := ioutil.ReadFile(path.Join(usrpath, kernelRelease, "modules.builtin"))
```

Comment:
On debian systems modules.builtin doesn't exist at least on the 4.19 kernels it does however exist on  the 5.3 packaging. So the code should handle that properly (e.g. assume nothing is builtin like its' done now)

Reply:
In any case, we’re unconditionally copying `modules.builtin` below, so the case when it doesn’t exist has already *not* been covered.